extensions/3DTILES_metadata/0.0.0/schema/3DTILES_metadata_implicit_tiling.schema.json

```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "title": "3DTILES_metadata extension for implicit tiling subtree metadata availability.",
+  "type": "object",
+  "properties": {
+    "class": {
+      "description": "The class referring back to a class definition in 3DTILES_metadata in tileset.json",
+      "type": "string"
+    },
+    "additionalProperties": {
+      "$ref": "3DTILES_metadata_implicit_tiling_object.schema.json"
+    }
```

Comment:
@lilleyse I talked with @ErixenCruz about this. we clarified the syntax for `allOf`, however, we had a question, so we didn't commit any changes yet.

Does this extension directly depend on `3DTILES_metadata`? if so, can we reuse the `instanceTable.property.schema.json` and just mention in the spec that JSON encoding is forbidden? 

If not, should we copy in the schema file and just remove the parts we don't use? Not sure if we should use the same filename or not.

Reply:
Yeah it depends on `3DTILES_metadata`, and you can copy `instanceTable.property.schema.json` in the same way that was done for `EXT_feature_metadata` (I forget the specifics, but we would disallow JSON encoding here as well)

Reply:
See https://github.com/CesiumGS/3d-tiles/pull/440#issuecomment-736117919. I think it actually depends on `3DTILES_tile_metadata`, which depends on `3DTILES_metadata`